static/temp/foods.json

```diff
@@ -0,0 +1,340 @@
+[
+  {
```

Comment:
We deleted this file, why are we adding it back?

Reply:
I just remembered that this component was created from the recnetly view task component and the old code were still there. Fixed anyways.